src/test/HostActivationTests/PortableAppActivation.cs

```diff
@@ -294,6 +296,81 @@ public void Framework_Dependent_AppHost_Succeeds()
                 .And.HaveStdOutContaining($"Framework Version:{sharedTestState.RepoDirectories.MicrosoftNETCoreAppVersion}");
         }
 
+        [Fact]
+        public void Framework_Dependent_AppHost_From_Global_Registry_Location_Succeeds()
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                return;
+            }
+
+            var fixture = sharedTestState.PreviouslyPublishedAndRestoredPortableTestProjectFixture
```

Comment:
You'll need to fix this after rebase with Elinor's renames...

Reply:
Yes, I saw this later...